exaudfclient/base/javacontainer/script_options/converter_legacy.cc

```diff
@@ -16,10 +16,19 @@ void ConverterLegacy::convertExternalJar(const std::string& value) {
     std::string jar;
 
     while (std::getline(stream, jar, ':')) {
-        m_jarPaths.insert(jar);
+        if (m_jarPaths.find(jar) == m_jarPaths.end()) {
```

Comment:
For what do you need the if

Reply:
You are right, if the element is already in the set, the std::insert does nothing.